src/utils/dates.js

```diff
@@ -10,3 +10,16 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+/**
+ * @param {TimeStamp} dateLastPurchased Firebase TimeStamp type from data
+ * @returns {boolean} whether date is within past 24 hours from current date
+ * transform TimeStamp and current date to milliseconds and check whether less than one day in milliseconds
+ */
+export function withinTwentyFourHours(dateLastPurchased) {
+	if (dateLastPurchased === null) return false;
+
+	const prevDate = dateLastPurchased.toDate().getTime();
+	const today = new Date().getTime();
+	return today - prevDate < ONE_DAY_IN_MILLISECONDS;
+}
```

Comment:
This is awesome!
Love the use of JSDoc tags, great practice especially for any future use with typescript.
Helps confirm my understanding of the function. 

awesome modular function, great checks and conversions done for the return.